server/models/review_model.py

```diff
@@ -25,9 +26,17 @@ def assign_reviewee(cls, review_id, user_id):
     @classmethod
     def update_status(cls, review_id, status):
         review = ReviewModel.get_review(review_id)
+        if(status == "pending"):
+            review.reviewer_id = None
         review.status = status
         db.session.commit()
 
+    @classmethod
+    def link_message_id(cls, review_id):
+        review = ReviewModel.get_review(review_id)
+        review.message = review_id
+        db.session.commit()
```

Comment:
Won't this always be the same as the id field? I don't think there is a purpose for the message column